lib/src/showcase.dart

```diff
@@ -641,11 +655,60 @@ class _ShowcaseState extends State<Showcase> {
             descriptionTextDirection: widget.descriptionTextDirection,
             toolTipSlideEndDistance: widget.toolTipSlideEndDistance,
             toolTipMargin: widget.toolTipMargin,
+            tooltipActionConfig: _getTooltipActionConfig(),
+            tooltipActions: _getTooltipActions(),
           ),
         ],
       ],
     );
   }
+
+  List<Widget> _getTooltipActions() {
+    final showCaseState = ShowCaseWidget.of(context);
```

Comment:
use context check before this

Reply:
It is used in the build method to provide action for the widget, eliminating the need for a context check since the context will always be mounted in the build method.